zql/zql.peg

```diff
@@ -118,12 +118,12 @@ searchPred
       RETURN(makeCompareField("in", f, v))
     }
   / v:searchValue {
-      INIT_ASSIGN_VAR(ss, makeSearchString(v))
+      INIT_ASSIGN_VAR(ss, makeCompareAny("srch", true, v))
```

Comment:
I'd like to buy a vowel (or two) please...

Reply:
added.